deploy/warehouse/target/data.sql

```diff
@@ -48,4 +120,15 @@ insert into warehouse.target (identifier, organism_id, control)
         set organism_id = EXCLUDED.organism_id
 ;
 
+
+insert into warehouse.target (identifier, control)
+    values
+        ('Hs04930436_g1',   't'),
+        ('Ac00010014_a1',   't')
+
+    on conflict (identifier) do update
+        set identifier  = coalesce(target.identifier, '')   || EXCLUDED.identifier,
```

Comment:
This line doesn't make much sense. The coalesce() is unnecessary because target.identifier is required to be not null by a column constraint and is also the conflicting value named in the `on conflict (identifier)` clause. Additionally, `||` is the concatenation operator (not a logical or), so this line would update the identifier to be doubled. The setting of `identifier` can be entirely removed from this upsert.